src/payment/strategies/adyenv2/adyenv2-initialize-options.ts

```diff
@@ -30,7 +30,7 @@ export default interface AdyenV2PaymentInitializeOptions {
      * @deprecated
      * Use additionalActionOptions instead as this property will be removed in the future
      */
-    threeDS2Options: AdyenThreeDS2Options;
+    threeDS2Options: ThreeDS2Options;
```

Comment:
as David's suggested, can you keep the prefix `Adyen` for this? Specially because it's deprecated and some clients could be consuming this type, we might be better off keeping this as it was, until we can remove all deprecated stuff.